core/src/main/scala/kafka/controller/ControllerChannelManager.scala

```diff
@@ -125,7 +126,8 @@ class ControllerChannelManager(controllerContext: ControllerContext, config: Kaf
         "controller-channel",
         Map("broker-id" -> broker.id.toString).asJava,
```

Comment:
We should take the chance and replace `broker.id.toString` with `broker.idString`.